chain/chaintest/action_test_helpers.go

```diff
@@ -81,3 +82,20 @@ func (test *ActionTest) Run(ctx context.Context, t *testing.T) {
 		}
 	})
 }
+
+// RunBenchmark executes the [ActionTest] and make sure all the benchmark assertions pass.
+func (test *ActionTest) RunBenchmark(ctx context.Context, b *testing.B) {
+	b.Run(test.Name, func(b *testing.B) {
```

Comment:
Can we get rid of `b.Run` here? This is typically used to run subtests / sub-benchmarks, but we're not doing that in this case.